test/e2e-tests.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs the end-to-end tests for the async component.
+
+# TODO explain how to use this locally
```

Comment:
Are we running locally or not?

Reply:
Local is an overloaded term here.  It runs locally within the test cluster.  This note is a reminder to add instructions for users who want to run the test on their local machine.  This will likely become a second script that assumes the user has a kubernetes cluster and (optionally?) pre installed knative serving/eventing/istio and the option to use a cloud based version of redis.